docs/docs/01-ibc/01-overview.md

```diff
@@ -160,34 +160,6 @@ Proofs are passed from core IBC to light clients as bytes. It is up to light cli
 [ICS-24 Host State Machine Requirements](https://github.com/cosmos/ics/tree/master/spec/core/ics-024-host-requirements).
 - The proof format that all implementations must be able to produce and verify is defined in [ICS-23 Proofs](https://github.com/cosmos/ics23) implementation.
 
-### [Capabilities](https://github.com/cosmos/cosmos-sdk/blob/main/docs/learn/advanced/10-ocap.md)
```

Comment:
The `Ports` section underneath this one references capabilities:

> Since IBC is designed to be secure with mutually distrusted modules operating on the same ledger,
binding a port returns a dynamic object capability. In order to take action on a particular port

I don't think the above statement is necessarily true in the context of ibcgo as a cosmos-sdk module. 

I think we can merged this with general removal of capabilities sections and address the more granular docs changes in another PR to finally close it out

Reply:
SGTM thanks for checking